selenium_tests/course_info/course_info_base_test_case.py

```diff
@@ -51,6 +51,9 @@ def setUpClass(cls):
         cls.people_page = CoursePeoplePageObject(cls.driver)
         cls.search_page = CourseSearchPageObject(cls.driver)
 
+        cls.test_data = settings.SELENIUM_CONFIG[
+            'course_info_tool']['test_data_for_multiple_users_add']
```

Comment:
Does it make sense for the entire course info base test case to use `test_data_for_multiple_users_add` as its test data? Seems like that's very specific.